liquibase-service/src/main/resources/db/odse/010-sp_investigation_event-001.sql

```diff
@@ -0,0 +1,2650 @@
+CREATE OR ALTER PROCEDURE dbo.sp_public_health_case_fact_datamart_event @phc_id_list nvarchar(max)
```

Comment:
The investigation stored proc is not called here but it can be taken up in a different PR